lib/rules/indent.js

```diff
@@ -160,9 +155,9 @@ class BinarySearchTree {
      * @returns {{key: number, value: *}|null} The found entry, or null if no such entry exists.
      */
     findLe(key) {
-        const iterator = this._rbTree.le(key);
+        const iterator = this._rbTree.reverseLowerBound(key);
 
-        return iterator && { key: iterator.key, value: iterator.value };
+        return { key: iterator.pointer[0], value: iterator.pointer[1] };
```

Comment:
In the comments, It may be empty, but I didn't find `null` in the test.

I mean if I need to judge the iterator with rbTree end here? This may cause an access error.

Reply:
I'd prefer keep the old behaviour - the rule has too many edge cases, so likely not covered by the tests.

Reply:
Ok, I'll change to to keeping the old behaviour.

Reply:
I went to look the code of functional-red-black-tree and it seems that it returns undefined here, so I return an empty Object.